src/tests/multihost/alltests/conftest.py

```diff
@@ -1057,6 +1057,59 @@ def remove_journalsssd():
     request.addfinalizer(remove_journalsssd)
 
 
+@pytest.fixture(scope='class')
+def sssdproxyldap_test(session_multihost, request):
+    """ Configure  sssdproxyldap
+        Configure  sssd.conf
+        Configure  nslcd.conf
+        Transport sssdproxyldap.sh to client machine
+        configure password for ldap user
+    """
+    tools = sssdTools(session_multihost.client[0])
+    execute_cmd(multihost, "yum install -y nss-pam-ldapd")
```

Comment:
I guess this sould be `session_multihost` instead of just `multihost`.

Reply:
done